src/app/service/BookService.java

```diff
@@ -24,5 +24,8 @@ public boolean bookExists(String name, String author, int year) {
         return book != null;
     }
 
-
+    public Book foundBook(String name,String author,int year){
```

Comment:
Maybe `findBook` ? also this seems to be returning a Book so the proper name can be `getBook`.
If not, make it return a `boolean`